plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2038,178 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class CollisionFrequencies:
+    @validate_quantities(
+        v_a={"none_shall_pass": True},
+        T_a={
+            "none_shall_pass": True,
+            "can_be_negative": False,
+            "equivalencies": u.temperature_energy(),
+        },
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    @particles.particle_input
+    def __init__(
+        self,
+        test_particle: particles.Particle,
+        field_particle: particles.Particle,
+        *,
+        v_a: u.cm / u.s = None,
+        T_a: u.K = None,
+        n_b: u.cm**-3,
+        T_b: u.K,
+        Coulomb_log: u.dimensionless_unscaled,
+    ):
+        r"""
+        Compute collision frequencies between test particles (labeled 'a') and field particles (labeled 'b')
+
+        Parameters
+        ----------
+        test_particle : |Particle|
```

Comment:
```suggestion
        test_particle : `~plasmapy.particles.particle_class.Particle`
```

Reply:
Is there a reason for typing out the whole reference as opposed to using the substitution?

Reply:
Oh, is that what that does? I've never seen that before.